lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -533,6 +536,7 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
         this.showingSearchResults = false;
         this.infiniteScroll = false;
         this.breadcrumbFolderTitle = null;
+        this.breadcrumbFolderTitleEvent.emit(this.breadcrumbFolderTitle);
```

Comment:
we could add a unit test that checks: When the folder is changed the new event emitter emits the value that it should, in this case the folder title